packages/salesforcedx-test-utils-vscode/src/testrunner.ts

```diff
@@ -1,19 +1,11 @@
-/*
- * rewritten test runner for integrating code coverage;
- * https://github.com/codecov/example-typescript-vscode-extension
- */
 'use strict';
 
-import * as fs from 'fs';
 import * as glob from 'glob';
 import * as paths from 'path';
-import { isNullOrUndefined } from 'util';
 
 // tslint:disable:no-var-requires
-const istanbul = require('istanbul');
 // tslint:disable-next-line:variable-name
 const Mocha = require('mocha');
-const remapIstanbul = require('remap-istanbul');
```

Comment:
So this is the custom test runner used for integration tests.   It is....complicated and more recent versions of mocha looks like it's not overly common to use this pattern anymore.  I just removed all the pieces related to code coverage.   

Note the replacement for istanbul (nyc) doesn't event have the same code based options and is only a command line client.  I attempted to pull it in outside of the int test run and it worked for me locally, but failed in GHA.